### PR TITLE
ML Translation

### PR DESCRIPTION
**Pull request by ellikap6**
uncompleted, but I want to push what I have so far

### CODE REVIEW

**Comment by tsalako** [comment]
Code looks good. Can you add a screenshot or a demo of some sort?

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ function showMessageFormIfViewingSelf() {
 /** Fetches messages and add them to the page. */
 function fetchMessages() {
   const url = '/messages?user=' + parameterUsername;
+  const parameterLanguage = urlParams.get('language');
+  let url = '/messages?user=' + parameterUsername;
```
Comment (tsalako) [inline src/main/webapp/js/user-page-loader.js:53]: is there a reason you are setting this twice?

Comment (ellikap6) [reply src/main/webapp/js/user-page-loader.js:53]: i got rid of the redundancy

**Comment by ellikap6** [comment]
Screenshot (25)

**Review by tsalako — APPROVED** [review]
Looks good! Feel free to merge.

**Pull request merged**